src/ynEIGEN/EigenStrategyManager.sol

```diff
@@ -100,6 +113,14 @@ contract EigenStrategyManager is
     IERC20 public oETH;
     IERC20 public stETH;
 
+    IRedemptionAssetsVaultExt public redemptionAssetsVault;
+    IWrapper public wrapper;
+
+    // @todo - struct
```

Comment:
agreed struct also:

we could do the following:

strategiesBalance[strategy] = _strategiesBalance;
        strategiesWithdrawalQueueBalance[strategy] = _strategiesWithdrawalQueueBalance
        
        
        are stored as a sum. of the 2 things since they get converted together 
using the Wrapper. This can be stored as  as a uint128.
        
        strategiesWithdrawnBalance gets stored as the other uint128.
        
        Thus both packed in the same slot. 
        
        Make sure when writing that the slot gets written exactly one time (do the update of both at the same time).

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/164/commits/9d06c2919998241fc79953c03598e17c6b3627d5